tests/perfclient.py

```diff
@@ -80,9 +72,11 @@ def add(parser):
             required=True,
         )
 
-    def get_command(host_arg, port_arg):
-        return ["host:", host_arg, "port:", port_arg]
+    args, unknown_args = cli_args(add, accept_unknown=True)
+
+    unknown_args = [term for arg in unknown_args for term in arg.split(" ")]
 
-    args = cli_args(add)
+    def get_command(*args):
+        return [*args] + unknown_args
 
-    infra.runner.run(args.build_dir, args.package, get_command, args)
+    infra.runner.run(args.build_dir, get_command, args)
```

Comment:
Was `args.package` unused?

Reply:
More than unused, it was removed. `runner.run` only takes 3 arguments, so this didn't work. But previously we had no tests calling `perfclient` directly.